resources/remove-sa.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SERVICE="${1}"
+if [ X"${SERVICE}" = X"" ]; then
+    echo "usage remove-sa.sh servicename"
+    exit 1
+fi
+
+COLLATION="${2:-}"
```

Comment:
COLLATION is not used and seems unneccesary. What is it's use?

Reply:
I agree. Removed it.